### PR TITLE
Add usage of ti.exe on Windows

### PR DESCRIPTION
**Pull request by xumingkuan**
Refer to https://github.com/taichi-dev/taichi/issues/435#issuecomment-586668275.
Shall we update the doc explicitly with "RelWithDebInfo", or there may be other build options?

### CODE REVIEW

**Comment by yuanming-hu** [comment]
Cool! Maybe https://stackoverflow.com/questions/7747857/in-cmake-how-do-i-work-around-the-debug-and-release-directories-visual-studio-2 is worth checking out?

File: cmake/TaichiMain.cmake [hunk 1]
```diff
@@ -5,4 +5,6 @@ if (WIN32)
     set_target_properties(ti PROPERTIES RUNTIME_OUTPUT_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/bin")
     set_target_properties(ti PROPERTIES RUNTIME_OUTPUT_DIRECTORY_DEBUG "${CMAKE_CURRENT_SOURCE_DIR}/bin")
     set_target_properties(ti PROPERTIES RUNTIME_OUTPUT_DIRECTORY_RELEASE "${CMAKE_CURRENT_SOURCE_DIR}/bin")
+    set_target_properties(ti PROPERTIES RUNTIME_OUTPUT_DIRECTORY_MINSIZEREL "${CMAKE_CURRENT_SOURCE_DIR}/bin")
+    set_target_properties(ti PROPERTIES RUNTIME_OUTPUT_DIRECTORY_RELWITHDEBINFO "${CMAKE_CURRENT_SOURCE_DIR}/bin")
```
Comment (yuanming-hu) [inline cmake/TaichiMain.cmake]: Now I see what happened :-)

**Comment by xumingkuan** [comment]
> Cool! Maybe https://stackoverflow.com/questions/7747857/in-cmake-how-do-i-work-around-the-debug-and-release-directories-visual-studio-2 is worth checking out?

It works.

Shall we assume that `%TAICHI_REPO_DIR%\bin` is already in the PATH? Then there will be no difference between Windows and other platforms.

**Comment by yuanming-hu** [comment]
Do we still have to add `taichi/bin` to `PATH`?

**Comment by xumingkuan** [comment]
> Do we still have to add `taichi/bin` to `PATH`?

Yes. I just removed it from the PATH and cmd becomes unable to recognize `ti`.

**Comment by yuanming-hu** [comment]
It's so great to have `ti` working on all platforms. Great job, Mingkuan!

**Comment by xumingkuan** [comment]
`ti` failed on my computer today, and I fixed it by https://superuser.com/questions/1437590/typing-python-on-windows-10-version-1903-command-prompt-opens-microsoft-stor and rebuilding.

**Pull request merged**